spec/features/happiness_exercises_spec.rb

```diff
@@ -103,4 +103,9 @@ def why_do_exercises
 
     expect(happiness_exercises).to have_previous_responses
   end
+
+  # scenario 'view happiness graph' do
```

Comment:
@a-ashbeck is this unnecessary or pending? if its pending uncomment and switch `scenario` to `skip` and put a comment above as to why its being skipped